### PR TITLE
Robert allen

### PR DESCRIPTION
**Pull request by robertjallen**

### CODE REVIEW

File: assignments/array-methods.js [hunk 1]
```diff
@@ -56,28 +56,35 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
...
+let largeShirts = runners.filter((runner) => runner.shirt_size === "L");
 console.log(largeShirts);
 
 // ==== Challenge 4: Use .reduce() ====
 // The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
-let ticketPriceTotal = [];
+let ticketPriceTotal = runners.reduce((total, runner) => total += runner.donation, 0);
 console.log(ticketPriceTotal);
 
 // ==== Challenge 5: Be Creative ====
```
Comment (AislynnEdmiston) [inline assignments/array-methods.js:83]: This is required to meet mvp. Please complete this

**Review by AislynnEdmiston — CHANGES_REQUESTED** [review]
Please finish Array methods. Then send me a dm please

File: assignments/array-methods.js (hunk 1 above)
Comment (robertjallen) [reply assignments/array-methods.js:83]: resolved challenge 5

File: assignments/array-methods.js [hunk 2]
```diff
@@ -48,36 +48,66 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
...
 console.log(largeShirts);
 
 // ==== Challenge 4: Use .reduce() ====
 // The donations need to be tallied up and reported for tax purposes. Add up all the donations into a ticketPriceTotal array and log the result
-let ticketPriceTotal = [];
+let ticketPriceTotal = runners.reduce((total, runner) => total += runner.donation, 0);
 console.log(ticketPriceTotal);
 
+
 // ==== Challenge 5: Be Creative ====
```
Comment (AislynnEdmiston) [inline assignments/array-methods.js:83]: Good Job!

**Pull request merged**